### PR TITLE
Upload logo

### PR DESCRIPTION
**Pull request by nebunez**
Closes https://github.com/owncast/owncast/issues/716

### CODE REVIEW

File: styles/config-public-details.scss [hunk 1]
```diff
@@ -74,3 +67,22 @@
...
+  img.logo-preview {
+    min-height: 120px;
+    min-width: 120px;
+    max-height: 256px;
+    max-width: 256px;
+    margin-right: 1em;
+    display: inline-block;
+    border: 1px solid var(--white-25);
+  }
+  .submit-button {
```
Comment (gingervitis) [inline styles/config-public-details.scss:85]: we can get rid of this style :)

**Comment by gabek** [comment]
When this gets merged in, please also merge https://github.com/owncast/owncast/pull/773, as they have to go in together.

**Review by gingervitis — APPROVED** [review]
Now that 0.0.6 is alive, let's merge this in for 0.0.7! 
I left a small comment, otherwise click the button whenever!

**Comment by gabek** [comment]
@nebunez Feel free to merge this and https://github.com/owncast/owncast/pull/773 whenever you're ready!

**Pull request merged**